.gitignore

```diff
@@ -1,6 +1,10 @@
+# don't pin versions
```

Comment:
I don't think comments in `.gitignore` are necessary. Would you mind removing them?